tests/apichecks/crds_test.go

```diff
@@ -23,7 +23,14 @@ import (
 
 	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/crd/crdloader"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test"
+	testcontroller "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test/controller"
+	testgcp "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test/gcp"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test/resourcefixture"
+	testvariable "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/test/resourcefixture/variable"
+	"github.com/ghodss/yaml"
```

Comment:
Please (always) use `sigs.k8s.io/yaml`.  I know there are some places we don't, but I consider those problematic

Reply:
Yes agreed; I tried using that one first but I was having issues unmarshalling to yaml;

i can investigate why some more and can probably take it out in places like these too: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/cd58b1e916a87a22686fa361f0166833f0fec7ac/pkg/test/unstructured.go#L20

Reply:
Ah figured it out 🙃 ; will create an issue and fix it elsewhere too (eventaully)